django_cloud_deploy/cloudlib/_utils.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper file for common functionality within cloublib."""
+
+
+def set_user_agent(http, user_agent):
+    """Set the user-agent on every request.
+
+    Args:
+       http - An instance of google_auth_httplib2.AuthorizedHttp.
+       user_agent: string, the value for the user-agent header.
+
+    Returns:
+       A modified instance of http that was passed in.
```

Comment:
This is a convention in Python that mutation methods do *not* return what the mutate.

For example:
`sort()` does a mutation and returns ` None` but `sorted()` returns a copy and returns that copy.

Reply:
ack